fourkitchens/suhumsci
# SHS-4966: Fix smart link popup/modal sizing and orientation issue

# READY FOR REVIEW

## Summary
- Limit size of autocomplete options in the ckeditor link dialog

## Urgency
- normal

## Steps to Test
1. Create a flexible page. Add some text in the main region text area
2. Select some text and add a link. Start typing some text in the Link URL fix and wait for the autocomplete options to appear. Confirm that the dialog is always smaller than the browser window.

**NOTE:** This PR partially fixes [the ticket](https://app.clickup.com/t/36718269/SHS-4966). The second acceptance criteria (prevent the dialog to jump when the user scrolls) is not possible without modifying the linkit module (and possible ckeditor itself), because the position adjustment on scroll is the default behavior of the link dialog.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
